gef-ui/templates/partials/facility-confirm-deletion.njk

```diff
@@ -0,0 +1,37 @@
+{% extends "index.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block pageTitle %}Confirm that you want to delete {{ heading }} facility 1?{% endblock %}
```

Comment:
Should this be
```
delete {{ heading }} facility {{ facilityId}} ?
```

instead of "facility 1"?

Reply:
Good catch